app/containers/EventsPage/Loadable.js

```diff
@@ -1,6 +1,6 @@
 /**
  *
- * Asynchronously loads the component for TobBar
+ * Asynchronously loads the component for EventsPage
```

Comment:
I think this comment was right.

Reply:
TopBar was the component we had created together and deleted. The component doesn`t exist anymore.